tools/etl/tg-jdbc-driver/README.md

```diff
@@ -8,18 +8,52 @@ The TigerGraph JDBC Driver is a Type 4 driver, converting JDBC calls directly in
 | --- | --- | --- | --- | --- | --- |
 | 1.0 | 2.2.4+ | 1.8 | Rest++ | JSON | Support builtin, compiled queries and loading jobs |
 | 1.1 | 2.4.1+ | 1.8 | Rest++ | ResultSet | Support tabular format and Spark |
-| 1.2 | 2.4.1+ | 1.8 | Rest++ | ResultSet | Support interpreted queries and Spark partitioning |
+| 1.2.x | 2.4.1~3.4 | 1.8 | Rest++ | ResultSet | Support interpreted queries and Spark partitioning |
 
 ## Dependency list
 | groupId | artifactId | version |
 | --- | --- | --- |
-| org.apache.commons | commons-io | 1.3.2 |
-| org.apache.httpcomponents | httpclient | 4.5.8 |
+| org.apache.commons | commons-io | 2.7 |
+| org.apache.httpcomponents | httpclient | 4.5.13 |
 | org.json | json | 20180813 |
 | org.glassfish | javax.json | 1.1.4 |
-| junit | junit | 4.11 |
+| org.junit.jupiter         | junit-jupiter-engine | 5.8.2 |
+| org.junit.vintage | junit-vintage-engine | 5.8.2 |
+| org.apache.logging.log4j | log4j-core | 2.17.2 |
+| org.apache.spark | spark-core_2.13 | 3.2.0 |
+
+## Download from Maven Central Repository
+
+Tigergraph JDBC Driver can be searched in https://search.maven.org as `tigergraph-jdbc-driver`.
+
+Please refer to [Versions compatibility](#versions-compatibility) to find the **suitable version**.
+
+* Use it in your java code:
+
+  * Apache Maven
+
+    ```xml
+    <dependency>
+      <groupId>com.tigergraph</groupId>
+      <artifactId>tigergraph-jdbc-driver</artifactId>
+      <version>{SUITABLE_VERSION}</version>
+    </dependency>
+    ```
+
+  * Gradle Groovy DSL
+
+    ```groovy
+    implementation 'com.tigergraph:tigergraph-jdbc-driver:{SUITABLE_VERSION}'
+    ```
+
+  Your build automation tool will download it autometically from maven central repository.
+
+* Use the jar file directly (e.g. Spark):
+
+  Search for `tigergraph-jdbc-driver` in https://search.maven.org to get the jar file.
```

Comment:
How about this?
`Go to [maven.org](https://search.maven.org/artifact/com.tigergraph/tigergraph-jdbc-driver) to get the jar file.`